src/main/java/uk/gov/hmcts/divorce/divorcecase/tab/CaseTypeTab.java

```diff
@@ -108,7 +108,8 @@ private void buildAosTab(ConfigBuilder<CaseData, State, UserRole> configBuilder)
             .field("applicant2SolicitorRepresented", NEVER_SHOW)
             .field("statementOfTruth")
             .field("applicant2StatementOfTruth", "statementOfTruth!=\"*\"")
-            .field("dateAosSubmitted");
+            .field("dateAosSubmitted")
+            .field("aosIsDrafted", NEVER_SHOW);
```

Comment:
is there a reason to add aosIsDrafted this as hidden ?

Reply:
I did not think there was a need to display this to the user.  I only needed it here so that the event show condition worked correctly in the dropdown